service/src/main/kotlin/io/provenance/api/frameworks/web/external/provenance/ProvenanceApi.kt

```diff
@@ -352,7 +382,10 @@ class ProvenanceApi {
             GET("/scope/query", handler::queryScope)
             POST("/verify", handler::verifyAsset)
             GET("/fees", handler::getFees)
-            PATCH("/permissions", handler::updateDataAccess)
+            "/permissions".nest {
+                PATCH("/data-access", handler::updateDataAccess)
```

Comment:
Nit: Dashes in endpoint URLs 🤮 I'd call "access" a bit redundant here and would have suggested just `/permissions/data`